src/Command/Dev/UpdateComposer/ClearModuleRequirements.php

```diff
@@ -44,55 +44,14 @@ public function __construct(
     /**
      * Generates script for clearing module requirements that run after composer install.
      *
-     * @param array $repos
-     * @return void
+     * @return string script name
+     *
+     * @throws \Magento\MagentoCloud\Filesystem\FileSystemException
      */
-    public function generate(array $repos)
+    public function generate()
     {
         $rootDirectory = $this->directoryList->getMagentoRoot();
-        $clearModulesFilePath = $rootDirectory . '/' . self::SCRIPT_PATH;
-        $stringRepos = var_export($repos, true);
-        $singlePackageType = ComposerGenerator::REPO_TYPE_SINGLE_PACKAGE;
-
-        $clearModulesCode = <<<CODE
```

Comment:
Moved to a static file. With new approach, no information about repos is necessary, so there are no variables in the script.